meganav.yaml

```diff
@@ -1,3 +1,699 @@
+            - title: QA
+              description: Test releases and features
+              url: https://discourse.ubuntu.com/c/quality/10
+            - title: Report Bugs
+              description: Report bugs on Launchpad
+              url: https://help.ubuntu.com/community/ReportingBugs
+            - title: Improve documentation
+              description: Help write the docs
+              url: https://wiki.ubuntu.com/DocumentationTeam/SystemDocumentation
+            - title: Local community teams
+              description: Join a community team
+              url: http://loco.ubuntu.com/teams/
+            - title: Donate
+              description: Donate to the Ubuntu project
+              url: /download/desktop/thank-you
+    - title: Mission
+      primary_links:
+        - title: Resources
+          links:
+            - title: Our mission
+              description: What we stand for
+              url: /community/mission
+            - title: Community council
+              description: Governing body for the project
+              url: https://discourse.ubuntu.com/t/what-is-the-ubuntu-community-council/706
+            - title: Ubuntu code of conduct
+              description: Expectations of participants
+              url: /community/code-of-conduct
+            - title: Diversity policy
+              description: Our stance on Diversity
+              url: /community/diversity
+            - title: Project governace
+              description: How the project is governed
+              url: /community/governance
+      secondary_links:
+        - title: Links
+          links:
+            - title: Community Council Agenda
+              url: https://discourse.ubuntu.com/t/community-council-meeting-agendas-schedule/708
+            - title: Ubuntu Technical Board
+              url: https://wiki.ubuntu.com/TechnicalBoard
+            - title: Technical Board Agenda
+              url: https://wiki.ubuntu.com/TechnicalBoardAgenda
+    - title: Keep up to date
+      primary_links:
+        - title: Resources
+          links:
+            - title: Read the blogs
+              description: Canonical's blogs
+              url: /blog
+            - title: Weekly Newsletter
+              description: Sign up to our newsletter
+              url: https://wiki.ubuntu.com/UbuntuWeeklyNewsletter
+            - title: Mailing lists
+              description: All the public mailing lists
+              url: https://lists.ubuntu.com/
+            - title: Ubuntu Planet
+              description: Aggregated member blogs
+              url: http://planet.ubuntu.com/
+            - title: Teams
+              description: The teams in the project
+              url: https://wiki.ubuntu.com/Teams
+      secondary_links:
+        - title: Links
+          links:
+            - title: Become a blog contributor
+              url: https://community.ubuntu.com/t/call-for-contributors-to-the-ubuntu-weekly-newsletter/1542
```

Comment:
I get 'Oops! That page doesn’t exist or is private.' when trying to visit this page

Reply:
@cassiocassio any idea what this link should be? Is there a public one?